cmd/cone/token.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/conductorone/cone/pkg/client"
+	"github.com/conductorone/cone/pkg/output"
+
+	"github.com/spf13/cobra"
+)
+
+type Token struct {
+	AccessToken string `json:"access_token"`
+	TokenType   string `json:"token_type"`
+	Expiry      string `json:"expiry"`
+}
+
+func tokenCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "token",
+		Short: "",
+		RunE:  tokenRun,
+	}
+	addRawTokenFlag(cmd)
+	return cmd
+}
+
+func tokenRun(cmd *cobra.Command, args []string) error {
+	ctx, _, v, err := cmdContext(cmd)
+	if err != nil {
+		return err
+	}
+
+	clientId, clientSecret, err := getCredentials(v)
+	if err != nil {
+		return err
+	}
+
+	if len(args) != 0 {
+		usageErrorString := cmd.UsageString()
+		return fmt.Errorf("expected 0 arguments, got %d\n"+usageErrorString, len(args))
+	}
```

Comment:
You don't have to do this in this PR, but we should add this func and use it everywhere we validate args like this

```go
func validateArgLenth(expectedCount int, args []string, cmd *cobra.Command) error {
    if len(args) == expectedCount { 
        return nil
    }

    return fmt.Errorf("expected %d arguments, got %d\n%s", expectedCount, len(args), cmd.UsageString())
}
```

Then in the command bodies it'd just be

```go
if err := validateArgLenth(0, args, cmd); err != nil {
    return err
}
```